components/Admin/PlayerList.tsx

```diff
@@ -69,20 +79,23 @@ function PlayerList(props) {
         alert("It could not be deleted.");
       }
     } catch (error) {
-      window.location = "/500";
+      window.location.href = "/500";
     }
   };
 
   return (
     <Box sx={{ width: "100%" }}>
-      <div sx={{ width: "100%", mb: 2 }} style={{ backgroundColor: "#1f2434" }}>
+      <div
+        data-sx={{ width: "100%", mb: 2 }}
```

Comment:
what's `data-sx`?

Reply:
The sx prop is a shortcut for defining custom styles that has access to the theme in `@mui/system`.
data-* is needed for custom attributes, because Typescript complains that Property 'sx' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.

Reference: https://stackoverflow.com/questions/46215614/property-does-not-exist-on-type-detailedhtmlprops-htmldivelement-with-react

Reply:
I find it a bit surprise that `sx` was working on `div`, which is not a Mui component. `sx` is a prop that is created from Mui component. 
I don't know what's magic behind but it's better to not use `sx` on non Mui component, IMO. 

also reference: https://stackoverflow.com/a/73768249/949795